RecoParticleFlow/PFProducer/python/particleFlowEGamma_cff.py

```diff
@@ -23,14 +27,29 @@
 particleBasedIsolationTmp.valueMapElePFblockIso = cms.string("gedGsfElectronsTmp")
 
 egmPhotonIsolationCITK = _egmPhotonIsolationAOD.clone()
+egmElectronIsolationCITK = _egmElectronIsolationCITK.clone()
+egmElectronIsolationPileUpCITK = _egmElectronIsolationCITKPileUp.clone()
 
-for iPSet in IsoConeDefinitionsTmp:
+#change particleBasedIsolation object to tmp
+for iPSet in IsoConeDefinitionsPhotonsTmp:
```

Comment:
My comments were about IsoConeDefinitionsPhotonsTmp

The egmElectronIsolationCITK edited here is a local copy (clone) already, which doesn't need the extra deepcopy.